packages/gafl-webapp-service/src/pages/terms-and-conditions/__tests__/route.spec.js

```diff
@@ -14,13 +14,50 @@ jest.mock('../../../processors/mapping-constants.js', () => ({
     'salmon-and-sea-trout': 'salmon_and_sea_trout'
   }
 }))
+jest.mock('@defra-fish/business-rules-lib', () => ({
+  FULFILMENT_SWITCHOVER_DATE: '2024-05-30T23:00:00.000Z'
```

Comment:
This should be set to a different value than what's in business rules lib, otherwise what's the point of mocking it? When you mock it, it's supposed to show that it's decoupled, which setting to the same value won't do.